src/python/lambda_pre_auth_user_expiry.py

```diff
@@ -0,0 +1,84 @@
+import os
+import datetime
+import dateutil
+from dateutil.relativedelta import relativedelta
+import boto3
+from boto3.dynamodb.conditions import Key
+
+def initializeDynamoTableResource():
```

Comment:
In python you don't have to initialise resources like this.
A simple 
`dynamodb = boto3.resource("dynamodb", region_name=regionName)`
`table = dynamodb.Table(tableName)` 
would work. 
Any variables/resources declared below the imports are global

Reply:
Ah, although if the tableName is dynamic then yes - I can see why you are doing this.

Reply:
This is done.